docs/src/main/asciidoc/vertx-reference.adoc

```diff
@@ -828,7 +824,7 @@ To better understand, let's detail how the HTTP request/response has been handle
 5. Once the reply is received by the sender, the content is written to the HTTP response
 
 
-=== Bidirectional communication with browsers using SockJS
+=== Bidirectional Communication with Browsers using SockJS
```

Comment:
Are you sure that we need to capitalize the titles? I thought we wanted the opposite but I haven't followed things that closely in this area.

/cc @michelle-purcell

Reply:
I was confused by the rule too, but it seems that Vale likes it that way.

I had a few issues with words like Vert.x (asked for an uppercase X which would be wrong) and APIs (asked for Apis)

Reply:
Ok, I fixed all the headers... Hopefully, Vale is not going to yell at me.